tests/v2/test_waku_store.nim

```diff
@@ -690,12 +736,16 @@ procSuite "Waku Store":
 
       let successResult = await proto3.resume(some(@[offListenSwitch.peerInfo.toRemotePeerInfo(),
                                                      listenSwitch.peerInfo.toRemotePeerInfo(),
-                                                     listenSwitch.peerInfo.toRemotePeerInfo()]))
+                                                     listenSwitch.peerInfo.toRemotePeerInfo(),
+                                                     listenSwitch2.peerInfo.toRemotePeerInfo()]))
       check:
-        proto3.messages.len == 10
+        proto3.messages.len == 14
```

Comment:
Wondering why 14? (ok, I found it :), `listenSweitch2` has 4 waku messages that `listenSwitch` does not have, though it would be good to add a comment and explain that, would be helpful for debugging)